modules/katamari/src/main/ts/ephox/katamari/api/Arr.ts

```diff
@@ -9,11 +9,9 @@ type ArrayGuardPredicate<T, U extends T> = (x: T, i: number) => x is U;
 type ArrayPredicate<T> = ArrayMorphism<T, boolean>;
 type Comparator<T> = (a: T, b: T) => number;
 
-/* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
So what happened to this rule? Is it smarter or was it removed?

Reply:
Not all of these methods were valid across browser when Katamari was first written. It's probably a very old `disable` block that was migrated through the years.

Reply:
Yes this rule was not reporting any problems so it was removed. Eslint v9 now can detect unused disable comments so they can be cleaned up.